apps/myaccount/src/components/shared/header.tsx

```diff
@@ -455,7 +433,6 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
                 actionIcon: <ArrowRightFromBracketIcon fill={ mode === "dark" ? "white" : "black" } />,
                 actionText: t("common:logout"),
                 menuItems: [
-                    resolveOrganizationLabel(),
```

Comment:
This should only be removed in the onprem product. Shall we use a config and conditionally hide it here?

Reply:
fixed